src/Gui/NaviCube.cpp

```diff
@@ -735,28 +783,28 @@ void NaviCubeImplementation::initNaviCube(QtGLWidget* gl) {
 	x = r45z * r54x * x;
 	z *= 1.46f; // corner face position
```

Comment:
Calculate from gap instead:
```suggestion
	z *= sqrt(3) * (1 - 4 * gap / 3); // corner face position
```

Reply:
Thanks, this is now in: commit f70329390cea